src/librustc_error_codes/error_codes/E0749.md

```diff
@@ -1,4 +1,19 @@
-Negative impls are not allowed to have any items. Negative impls
-declare that a trait is **not** implemented (and never will be) and
-hence there is no need to specify the values for trait methods or
-other items.
+An item was added on a negative impl.
```

Comment:
@JohnTitor I think the "item" here like you said means "type". Should we change it?
```suggestion
An associated type was added on a negative impl.
```

Reply:
Well, "item" means "associated items" so I don't think we should change it.